modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Secure Network Observability with a network policy using the web console
+You may need to create a network policy to secure ingress traffic to the 'netobserv' namespace. Using the web console, you can create a network policy 
```

Comment:
```suggestion
You might need to create a network policy to secure ingress traffic to the 'netobserv' namespace. Using the web console, you can create a network policy 
```

Reply:
Looks like the rest of the sentence is missing here. Or it needs to be something like, "You can create a network policy using the web console using the following procedure:"